offline_signing/transfer.sh

```diff
@@ -93,11 +93,41 @@ upload() {
     fi
 }
 
-roll() {
+ready() {
     local channel="$1"
     local version="$2"
     local board="$3"
 
+    echo "Rollout set to 0% for ${board}"
+    updateservicectl \
+        --server="https://public.update.core-os.net" \
+        --user="${ROLLER_USERNAME}" \
+        --key="${ROLLER_API_KEY}" \
+        group percent \
```

Comment:
Will this terminate any existing rollout in the group?

Reply:
yeah, part of the behavior of setting a percent anywhere is that it will pause the existing rollout. 

it's less termination and more pausing, it would be possible to unpause it (although in this specific case it doesn't make much sense). but, for instance, if someone changes the percent in the web ui at some point, that would also pause the automated rollout.

Reply:
Probably also needs a comment to that effect.